src/silvistat/storage.py

```diff
@@ -1,68 +1,96 @@
 import pdal
 import tiledb
 import numpy as np
+from math import floor
 
 from . import Bounds
 
 class Storage(object):
-
-    def __init__(self, dirname, filename, config=None, cell_size=30,
+    """ Handles storage of shattered data in a TileDB structure. """
+    def __init__(self, dirname, filename, config= None, cell_size=30,
                  chunk_size=16):
         self.dirname = dirname
         self.src_file = filename
         self.config = config
         self.cell_size = cell_size
```

Comment:
cell size can never change, right? Once the tdb is created, the cell size is fixed forever. This should be stored INSIDE the tdb as metadata and never provided by users or overridden in any way.